src/cloud/benchflow/data-analyses-scheduler/scripts/sparkArgsConstructors.go

```diff
@@ -11,6 +11,8 @@ import (
 func getConfigFilePath(SUTVersion string, SUTName string, SUTType string, fileName string) string {
 	// Split the version into 3 numbers (eg. 1.0.2 -> [1 0 2])
 	versionNums := strings.Split(SUTVersion, ".")
+	SUTName = strings.ToLower(SUTName)
```

Comment:
This assumes that all the folders and files in the plugin repo are lowercase. That is fine for now, for the sake of traceability, I took a note in https://github.com/benchflow/sut-plugins/issues/4